README.md

```diff
@@ -52,50 +43,38 @@ _NOTE:_
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
 might be per cluster/namespace/container for cloud native architecture.
 - Please refer structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
+- CCNP project collects container level primitives by implementing unified APIs defined in [CC Trusted API](https://github.com/cc-api/cc-trusted-api). The project will be moved to [CC Trusted API](https://github.com/cc-api/cc-trusted-api) in the near future. 
```

Comment:
The CCNP project collects container level primitives by implementing the unified APIs defined..